### PR TITLE
Add gif and fix alignment

### PR DESCRIPTION
**Pull request by cait1inhennessy**
Replace bar chart icon with animated gif from designers.  Also fix alignment.

Before: 
capture2

After: 
capture

### CODE REVIEW

File: webapp/src/serialindicator.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ export class SerialIndicator extends React.Component<SerialIndicatorProps, Seria
         return(
             <div title={lf("Open console")} className="ui label circular" tabIndex={0} onClick={this.props.onClick} onKeyDown={sui.fireClickOnEnter}>
                 <div className="detail">
-                    <sui.Icon icon="bar graph"/>
+                    <img className="barcharticon" src={Util.pathJoin(pxt.webConfig.commitCdnUrl, `images/Bars_black.gif`)}></img>
```
Comment (pelikhan) [inline webapp/src/serialindicator.tsx]: This image won't load in offline mode.

Comment (cait1inhennessy) [reply webapp/src/serialindicator.tsx]: How can we fix that?

Comment (pelikhan) [reply webapp/src/serialindicator.tsx]: maybe we just hide it if it fails to load

Comment (pelikhan) [reply webapp/src/serialindicator.tsx]: Open a bug to fix charting support when offline. This gif + smoothie need to be properly cached.

Comment (microsoftsam) [inline webapp/src/serialindicator.tsx]: Need alt text for the img.

Comment (microsoftsam) [reply webapp/src/serialindicator.tsx]: or aria-hidden

**Comment by cait1inhennessy** [comment]
@pelikhan @microsoftsam Ping for another review. I added alt text and offline caching.

File: webapp/src/serialindicator.tsx [hunk 2]
```diff
@@ -41,7 +41,7 @@ export class SerialIndicator extends React.Component<SerialIndicatorProps, Seria
         return(
             <div title={lf("Open console")} className="ui label circular" tabIndex={0} onClick={this.props.onClick} onKeyDown={sui.fireClickOnEnter}>
                 <div className="detail">
-                    <sui.Icon icon="bar graph"/>
+                    <img alt="Animated bar chart" className="barcharticon" src={Util.pathJoin(pxt.webConfig.commitCdnUrl, `images/Bars_black.gif`)}></img>
```
Comment (pelikhan) [inline webapp/src/serialindicator.tsx]: use ``lf`` to localize alt text

**Pull request merged**